### PR TITLE
Brittany Bug Fix

### PR DESCRIPTION
**Pull request by brittanykohler**
Fixed new/edit categories bug, wrote cucumber and rspec tests for fix.

### CODE REVIEW

File: features/support/paths.rb [hunk 1]
```diff
@@ -17,7 +17,8 @@ def path_to(page_name)
       '/'
     when /^the new article page$/
       '/admin/content/new'
-
+    when /^the categories page$/
+      '/admin/categories/new'
```
Comment (kariabancroft) [inline features/support/paths.rb:21]: This was a nice add to make sure the cucumber test was more readable

**Comment by kariabancroft** [comment]
Nice job on this - I don't think you want the files that are in the db/\* folder to have been checked in though?

**Comment by brittanykohler** [comment]
You are right-- I should probably add these to gitignore. Thanks!

**Comment by kariabancroft** [comment]
Yep! I'd also like to see them deleted from your PR so that in a real-world scenario, they wouldn't be merged into the baseline

**Comment by brittanykohler** [comment]
Files have been deleted! Thanks for pointing them out.

**Comment by kariabancroft** [comment]
Great! Ready to close!

**Pull request closed without merging**